src/libcore/convert.rs

```diff
@@ -380,7 +386,55 @@ pub trait TryInto<T>: Sized {
     fn try_into(self) -> Result<T, Self::Error>;
 }
 
-/// Attempt to construct `Self` via a conversion.
+/// Simple and safe type conversions that may fail in a controlled
+/// way under some circumstances. It is the reciprocal of [`TryInto`].
```

Comment:
This is a bit too much information for a summary line; they should be quite short. That said, this PR has already been merged; @icefoxen any interest in a follow-on?

Reply:
Not this time I'm afraid.  As suggested by how long it took me to notice this, life is pretty hectic at the moment.  :disappointed: